karate-core/src/main/java/com/intuit/karate/core/Step.java

```diff
@@ -219,6 +220,14 @@ public void setComments(List<String> comments) {
         this.comments = comments;
     }
 
+    public StepRuntime.MethodMatch getMatch() {
```

Comment:
this name can be better to make usage in engine more readable. perhaps getMatchingMethod ?